notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,581 @@
+{
```

Comment:
<p>I suggest checking the acceleration. Other people tried to extract it from the velocity and they had to apply an spline to the velocity before applying the gradient. I do not believe this will be critical for this test. Maybe we can even remove it. We might need to discuss more. Maybe we should ask in the SPA meeting? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I will pause the use of acceleration, after further discussion if it's needed.</p>